docs/src/main/paradox/client-side/host-level.md

```diff
@@ -191,6 +191,6 @@ the pool will "pull out" single requests when capacity is available on one of th
 @@@
 
 @@@ div { .group-java }
-For now, please see the Scala examples in @ref[Scala Host-Level Client API](../../../scala/http/client-side/host-level.md#examples).
+For now, please see the Scala examples in [Scala Host-Level Client API](../client-side/host-level.html#examples?language=scala).
```

Comment:
This might break the algolia search, AFAIK. We need to reconfigure where to read the language from, right?

Reply:
I'd say algolia could be language-agnostic now - but indeed we might have to do something to make them aware of the new locations of things? I'm not sure how that works?

Reply:
There is a JS snippet on the pages initializing the widget. Right now it determines the language by checking `window.location.pathname` if this contains `/java/` or `java.html`.
We need to check `window.location.search` instead and should contain `language=java`.
The other thing though, is how Algolia does the indexing. Because if I'm on the Java page and I search for something, I want to see directly the Java on my result, not the Scala one.

Reply:
paradox already remembers the 'current language' in a cookie, so it might not be necessary for algolia to be aware of the language (anymore)?

Reply:
Oh interesting point!